apps/codelab/src/app/admin/feedback/feedback-message-table/feedback-message-table.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, Input, ViewChild, Output, EventEmitter, OnInit } from '@angular/core';
+import { MatMenuTrigger } from '@angular/material';
+import { MatTableDataSource } from '@angular/material/table';
+import { MatSort } from '@angular/material/sort';
+import { Message } from '@codelab/feedback/src/lib/message';
+
+@Component({
+  selector: 'codelab-feedback-message-table',
+  templateUrl: './feedback-message-table.html',
+  styleUrls: ['./feedback-message-table.css']
```

Comment:
Consider using OnPush here?
https://netbasal.com/a-comprehensive-guide-to-angular-onpush-change-detection-strategy-5bac493074a4